packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -48,9 +48,12 @@ class Tooltip extends StatefulWidget {
   ///
   /// All parameters that are defined in the constructor will
   /// override the default values _and_ the values in [TooltipTheme.of].
+  ///
+  /// If [message] is `null`, the semantics label of the [child] will be used as
+  /// the string in the tooltip.
   const Tooltip({
     Key key,
-    @required this.message,
+    this.message,
```

Comment:
Maybe we should have a separate constructor for this new feature. It may be difficult to debug if you forget to specify the message and suddenly a random string from the semantics tree appears.